webapp/src/components/system_console/bot.tsx

```diff
@@ -221,6 +230,16 @@ const ServiceItem = (props: ServiceItemProps) => {
     const intl = useIntl();
     const hasAPIKey = type !== 'asksage';
     const isOpenAIType = type === 'openai' || type === 'openaicompatible' || type === 'azure';
+
+    const getDefaultMaxTokens = useCallback(() => {
```

Comment:
Not sure we need to bother with useCallback here. I think it won't do anything here?